### DISCUSSION TITLE
how to get gpu vram info?

### DISCUSSION THREAD

**Discussion by lzm420241** [gfx-rs/wgpu#4757] [General]
i need get gpu vram info, yes,by limits，i get some useful data, but any data don't equal to the info that system show.

**Comment by cwfitzgerald** [comment]
You currently can't. See https://github.com/gfx-rs/wgpu/issues/2447

**Reply by lzm420241** [reply]
thanks for you reply. will  it support in future? or when?

**Reply by cwfitzgerald** [reply]
The issue has a plan for implementation - as for when, as soon as someone cares enough to implement it :)
